src/docker/Dockerfile

```
@@ -1,4 +1,4 @@
-FROM node:14.16.1-alpine3.13
+FROM node:14.17.0-alpine3.13
```

I would change this to:
```suggestion
FROM node:14-alpine3.13
```

Which uses the latest `14.x` release to match what we check with on CI (for less surprises). Otherwise, we might run CI on a different NodeJS version (say `14.18.1` today) than the version we build our docker container with (here `14.17.0`).